src/python/pants/core/util_rules/environments.py

```diff
@@ -171,6 +179,10 @@ class DockerImageField(StringField):
         This value may be any image identifier that the local Docker installation can accept.
         This includes image names with or without tags (e.g. `centos6` or `centos6:latest`), or
         image names with an immutable digest (e.g. `centos@sha256:<some_sha256_value>`).
+
+        The choice of image ID can affect the reproducibility of builds. Consider using an
+        immutable digest if reproducibility is needed, but regularly ensure that the image
```

Comment:
what does the grammar of  "regularly ensure" mean? You're encouraging the user to do that?

Reply:
Yes. "Consider … but regularly ensure"